services/galaxy-toolshed/galaxy-toolshed-activity.service.js

```diff
@@ -0,0 +1,70 @@
+import Joi from 'joi'
+import { formatDate } from '../text-formatters.js'
+import { age as ageColor } from '../color-formatters.js'
+import BaseGalaxyToolshedService from './galaxy-toolshed-base.js'
+
+const repositoryRevisionInstallInfoSchema = Joi.array().items(
+  Joi.object({
+    create_time: Joi.string().required(),
+  }).required(),
+  Joi.object({}),
+  Joi.object({})
+)
+
+export default class GalaxyToolshedActivity extends BaseGalaxyToolshedService {
+  static category = 'activity'
+  static route = {
+    base: 'galaxy-toolshed/create-date',
+    pattern: ':repositoryName/:owner',
+  }
+
+  static examples = [
+    {
+      title: 'Galaxy Toolshed - Create Time',
+      namedParams: {
+        repositoryName: 'sra_tools',
+        owner: 'iuc',
+      },
+      staticPreview: this.render({
+        createDate: new Date(0).setUTCSeconds(1538288239),
+      }),
+    },
+  ]
+
+  static defaultBadgeData = {
+    label: 'create date',
+  }
```

Comment:
I realize that the underlying field name in the response is `create_time`, however, we're not obliged to mirror the wording of upstream providers. In this case I find `create date` to sound odd, and would suggest we use `release date` instead, which will be far more consistent with the other similar types of badges we provide for other services.

Reply:
The API grabs only the first date of the release, not the last. Should I change it anyway ?

Reply:
That was my understanding as well, and that's why I suggested `release date` as that's what is used for other badges (like the VS Marketplace). An alternative might to simply use `created`